radixdlt-core/radixdlt/src/integrationTest/java/com/radixdlt/integration/steady_state/simulation/full_function_forks/ShutdownForkSanityTest.java

```diff
@@ -0,0 +1,114 @@
+ * artefacts due to errors in transmission;
+ *
+ * C. bugs, hacks, logic errors or faults in the Work or any communication;
+ *
+ * D. system software or apparatus including but not limited to losses caused by errors
+ * in holding or transmitting tokens by any third-party;
+ *
+ * E. breaches or failure of security including hacker attacks, loss or disclosure of
+ * password, loss of private key, unauthorised use or misuse of such passwords or keys;
+ *
+ * F. any losses including loss of anticipated savings or other benefits resulting from
+ * use of the Work or any changes to the Work (however implemented).
+ *
+ * You are solely responsible for; testing, validating and evaluation of all operation
+ * logic, functionality, security and appropriateness of using the Work for any commercial
+ * or non-commercial purpose and for any reproduction or redistribution by You of the
+ * Work. You assume all risks associated with Your use of the Work and the exercise of
+ * permissions under this License.
+ */
+
+package com.radixdlt.integration.steady_state.simulation.full_function_forks;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import com.radixdlt.harness.simulation.NetworkLatencies;
+import com.radixdlt.harness.simulation.NetworkOrdering;
+import com.radixdlt.harness.simulation.SimulationTest;
+import com.radixdlt.harness.simulation.SimulationTest.Builder;
+import com.radixdlt.harness.simulation.application.RadixEngineUniqueGenerator;
+import com.radixdlt.harness.simulation.monitors.consensus.ConsensusMonitors;
+import com.radixdlt.harness.simulation.monitors.ledger.LedgerMonitors;
+import com.radixdlt.harness.simulation.monitors.radix_engine.RadixEngineMonitors;
+import com.radixdlt.mempool.MempoolConfig;
+import com.radixdlt.statecomputer.forks.ForksModule;
+import com.radixdlt.sync.SyncConfig;
+import java.time.Duration;
+import org.assertj.core.api.AssertionsForClassTypes;
+import org.junit.Test;
+
+public final class ShutdownForkSanityTest {
+  private final Builder bftTestBuilder;
+
+  public ShutdownForkSanityTest() {
+    bftTestBuilder =
+        SimulationTest.builder()
+            .numNodes(4)
+            .pacemakerTimeout(3000)
+            .networkModules(NetworkOrdering.inOrder(), NetworkLatencies.fixed())
+            .fullFunctionNodes(SyncConfig.of(400L, 10, 2000L))
+            .addRadixEngineConfigModules(new ShutdownForkModule(), new ForksModule())
+            .addNodeModule(MempoolConfig.asModule(1000, 10))
+            .addTestModules(
+                ConsensusMonitors.safety(),
+                ConsensusMonitors.noTimeouts(),
+                ConsensusMonitors.directParents(),
+                LedgerMonitors.consensusToLedger(),
+                LedgerMonitors.ordered(),
+                RadixEngineMonitors.noInvalidProposedCommands())
+            .addMempoolSubmissionsSteadyState(RadixEngineUniqueGenerator.class);
+  }
+
+  @Test
+  public void sanity_tests_should_pass() {
```

Comment:
Is it easy to add some sanity checks once the node has shutdown - eg:

* Testing construction throws?
* Check that the max transaction is the end of the last epoch (epoch 1?)
* Checking can still call transaction stream or other Core API endpoints (other than construction)